multisig/proposals/arbitrumsepolia/vip-004/index.ts

```diff
@@ -0,0 +1,85 @@
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { arbitrumsepolia } = NETWORK_ADDRESSES;
+
+const XVS_STORE = "0x3c49eB8FaDD3FdB0a4Bbb40F8A7e3ce821298607";
+export const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
+
+const vip004 = () => {
+  return makeProposal([
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "_acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: XVS_STORE,
+      signature: "acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "setAccessControl(address)",
+      params: [ACM],
+    },
```

Comment:
ACM already configured, we can remove this command

Reply:
Done https://github.com/VenusProtocol/vips/pull/250/commits/c90febea2255ecd32f1f7bc707fef9af85232b9f